src/client/ducks/fetch/selectors/index.js

```diff
@@ -0,0 +1,27 @@
+/* @flow */
+import { createSelector } from 'reselect'
+
+export type StatusType = {
+  isFetching: boolean,
+  error: Object
+}
+
+export type StateType = {
+  fetch: {
+    [key: string]: StatusType
+  }
+}
+
+export const fetchSelector = (state: Object): Object => state.fetch
+
+export const statusFetchFactory = (actionType: string): Function =>
+  createSelector(
+    fetchSelector,
+    (fetchState) => {
```

Comment:
Seems most places we do lambdas with one parameter we omit parentheses. I don't feel strongly either way, though, as long as we're consistent :)